src/features/filter-transactions/model.ts

```diff
@@ -1,32 +1,48 @@
-export type Status = 'committed' | 'rejected' | null;
-export type TabDefaultScreen = 'all' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
-export type TabBlocksScreen = 'transactions' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
-export type TabAssetsSort = 'recent' | 'value';
+import type { TransactionStatus } from '@/shared/api/schemas';
+
+export type Status = TransactionStatus | null;
+export type TabAccountTransactions = 'transactions' | 'instructions';
+export type TabInstructions =
+  | ''
```

Comment:
`''`?..

Makes sense move this into `api/schemas.ts`. It will be there later, and it belongs there. And no non-sensual blind `''`, please.

```suggestion
export type InstructionKind =
```